src/SDKs/SqlManagement/Management.Sql/Generated/SqlManagementClient.cs

```diff
@@ -422,19 +422,6 @@ public partial class SqlManagementClient : ServiceClient<SqlManagementClient>, I
         /// </summary>
         public virtual IManagedInstanceEncryptionProtectorsOperations ManagedInstanceEncryptionProtectors { get; private set; }
 
-        /// <summary>
-        /// Initializes a new instance of the SqlManagementClient class.
-        /// </summary>
-        /// <param name='httpClient'>
-        /// HttpClient to be used
-        /// </param>
-        /// <param name='disposeHttpClient'>
-        /// True: will dispose the provided httpClient on calling SqlManagementClient.Dispose(). False: will not dispose provided httpClient</param>
-        protected SqlManagementClient(HttpClient httpClient, bool disposeHttpClient) : base(httpClient, disposeHttpClient)
```

Comment:
Please update the changes introduced here to the `PackageReleaseNotes` in csproj

Reply:
This is also auto generated. Should I revert all the changes that are not tied to my feature?

Reply:
Please do not revert any files. 
Since autogenerating from the latest changes introduces these changes, they must be recorded in the `PackageReleaseNotes`
Please consult with @jaredmoo regarding this

Reply:
Weird, did autorest remove this constructor?

Reply:
Yes

Reply:
@fearthecowboy any idea why this might happen

Reply:
Which version of the c# generator is this using?

What happens when you generate with `--preview` ? 

G

Reply:
@fearthecowboy 
Doesn't look like it. Couldn't see it in the code gen script or the config file

Reply:
Can someone try with `--preview`?

Reply:
I can try it.

Reply:
@fearthecowboy Here is the PR which used preview version of autorest. https://github.com/Azure/azure-sdk-for-net/pull/5029

Reply:
Looks like that deletes the constructor too

Reply:
So, shall we get this merged? @dsgouda @jaredmoo

Reply:
I already signed off several days ago, I don't have the ability to merge.

Reply:
I wanted to make sure this doesn't adversely affect customers. SInce this is preview, it should be OK to merge this